exs/mundo_1/c/009.c

```diff
@@ -0,0 +1,35 @@
+/*
+    009 - Tabuada
+        Faça um programa que leia um número inteiro qualquer e mostre na tela a sua
+        tabuada
+*/
+
+#include <stdio.h>
+
+int main(void)
+{
+    int num;
+    printf("Digite o número que você quer ver a tabuada: ");
+    scanf("%i", &num);
+
+    printf("==================\n");
+    for (int i = 0; i <= 10; i++)
+    {
+        printf("%i x %i", num, i);
+        
+        // alinha os sinais de igualdade
+        if (i < 10)
+        {
+            printf("  ");
+        }
+        else
+        {
+            printf(" ");
+        }
+
+        printf("= %i\n", num * i);
+    }
```

Comment:
Esse exercício deve ser feito sem o uso de estruturas de repetição. 
```suggestion
    printf("%i x %i  = %i\n", num, 1, num * 1);
    printf("%i x %i  = %i\n", num, 2, num * 2);
    printf("%i x %i  = %i\n", num, 3, num * 3);
    printf("%i x %i  = %i\n", num, 4, num * 4);
    printf("%i x %i  = %i\n", num, 5, num * 5);
    printf("%i x %i  = %i\n", num, 6, num * 6);
    printf("%i x %i  = %i\n", num, 7, num * 7);
    printf("%i x %i  = %i\n", num, 8, num * 8);
    printf("%i x %i  = %i\n", num, 9, num * 9);
    printf("%i x %i = %i\n", num, 10, num * 10);
```